CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Here is a template for new release sections
 - pollutant and subclasses (#51)
 - structure of the ontology (#47)
 - change peat to solid and not gas (#39)
+- StateOfMatter and subclasses (#45)
```

Comment:
@akleinau - I did not indicate it clearly, but the convention I took was to indicate the PR number and not the issue number. In the changelog we want to track what was changed and where and not directly why (why is still accessible via the issue number which everyone should indicate in their PR :))

Reply:
I updated that in #70, so no need to change it for now , just a heads up for the future